tests/ClientTest.php

```diff
@@ -333,72 +185,22 @@ private function assertMixedValueAndArray($expected_value, $actual_value)
-    {
-        $reflection = new \ReflectionProperty(Client::class, 'severityMap');
-        $reflection->setAccessible(true);
-        $client = ClientBuilder::create()->getClient();
-
-        $predefined = [E_ERROR, E_WARNING, E_PARSE, E_NOTICE, E_CORE_ERROR, E_CORE_WARNING,
-                       E_COMPILE_ERROR, E_COMPILE_WARNING, E_USER_ERROR, E_USER_WARNING,
-                       E_USER_NOTICE, E_STRICT, E_RECOVERABLE_ERROR, ];
-        $predefined[] = E_DEPRECATED;
-        $predefined[] = E_USER_DEPRECATED;
-        $predefined_values = ['debug', 'info', 'warning', 'warning', 'error', 'fatal'];
-
-        // step 1
-        foreach ($predefined as &$key) {
-            $this->assertContains($client->translateSeverity($key), $predefined_values);
-        }
-        $this->assertEquals('error', $client->translateSeverity(123456));
-        // step 2
-        $client->registerSeverityMap([]);
-        $this->assertMixedValueAndArray([], $reflection->getValue($client));
-        foreach ($predefined as &$key) {
-            $this->assertContains($client->translateSeverity($key), $predefined_values);
-        }
-        $this->assertEquals('error', $client->translateSeverity(123456));
-        $this->assertEquals('error', $client->translateSeverity(123456));
-        // step 3
-        $client->registerSeverityMap([123456 => 'foo']);
-        $this->assertMixedValueAndArray([123456 => 'foo'], $reflection->getValue($client));
-        foreach ($predefined as &$key) {
-            $this->assertContains($client->translateSeverity($key), $predefined_values);
-        }
-        $this->assertEquals('foo', $client->translateSeverity(123456));
-        $this->assertEquals('error', $client->translateSeverity(123457));
-        // step 4
-        $client->registerSeverityMap([E_USER_ERROR => 'bar']);
-        $this->assertEquals('bar', $client->translateSeverity(E_USER_ERROR));
-        $this->assertEquals('error', $client->translateSeverity(123456));
-        $this->assertEquals('error', $client->translateSeverity(123457));
-        // step 5
-        $client->registerSeverityMap([E_USER_ERROR => 'bar', 123456 => 'foo']);
-        $this->assertEquals('bar', $client->translateSeverity(E_USER_ERROR));
-        $this->assertEquals('foo', $client->translateSeverity(123456));
-        $this->assertEquals('error', $client->translateSeverity(123457));
-    }
-
-    public function testSendChecksShouldCaptureOption()
+    public function testSendChecksBeforeSendOption()
     {
-        $shouldCaptureCalled = false;
+        $beforeSendCalled = false;
 
         $client = ClientBuilder::create([
             'dsn' => 'http://public:secret@example.com/1',
-            'should_capture' => function () use (&$shouldCaptureCalled) {
-                $shouldCaptureCalled = true;
+            'before_send' => function () use (&$beforeSendCalled) {
+                $beforeSendCalled = true;
 
-                return false;
+                return null;
```

Comment:
Since you're not returning the event, maybe you should assert that nothing is being sent too?